controller/api/models.py

```diff
@@ -519,7 +536,6 @@ class Push(UuidAuditedModel):
     class Meta:
         get_latest_by = 'created'
         ordering = ['-created']
-        unique_together = (('app', 'uuid'),)
```

Comment:
I totally agree that this bump should not change our schema definitions. I'll look for a way around this. :)